models/lorenz_attractor.nestml

```diff
@@ -0,0 +1,31 @@
+neuron lorenz_attractor:
+
+initial_values:
+  x real = 1
+  y real = 1
+  z real = 1
+end
+
+equations:
+  shape foo = exp(-t)
+  x' = sigma * (y - x) / s
+  y' = (x * (rho - z) - y) / s
+  z' = (x * y - beta * z) / s
+end
+
+update:
+  integrate_odes()
+end
+
+parameters:
+  sigma real = 10
+  beta real = 8/3
+  rho real = 28
+end
+
+input:
```

Comment:
does this need to be specified? (I guess so)
it's a bit ugly so it would be nice if we could remove it and make it empty by default

Reply:
@jougs fair point about it being a bad idea if it could happen by accident...